### PR TITLE
Re-add the db reset command.

### PR DESCRIPTION
**Pull request by jctanner**
```
(oci_env_tests) [jtanner@p1 oci_env]$ oci-env db reset
Name: django-extensions
Version: 3.2.0
Summary: Extensions for Django
Home-page: http://github.com/django-extensions/django-extensions
Author: Michael Trier
Author-email: mtrier@gmail.com
License: MIT License
Location: /usr/local/lib/python3.8/site-packages
Requires: Django
Required-by:

You have requested a database reset.
This will IRREVERSIBLY DESTROY
ALL data in the database "pulp".
Are you sure you want to do this?

Type 'yes' to continue, or 'no' to cancel: pulp [None]: root:INFO: Executing... "DROP DATABASE "pulp";"
pulp [None]: root:INFO: Executing... "CREATE DATABASE "pulp" WITH OWNER = "postgres"  ENCODING = 'UTF8';"
Reset successful.
#############################################################
WARNING: Please kill the compose stack and bring it up again.
#############################################################
(oci_env_tests) [jtanner@p1 oci_env]$ echo $?
0
```

Then the user needs to ctrl-c their instance of `oci-env up` and re-run it.

### CODE REVIEW

**Comment by dkliban** [comment]
I get the follwoing error.

```
[dkliban@localhost oci_env]$ oci-env db
Traceback (most recent call last):
  File "/home/dkliban/.local/bin/oci-env", line 33, in <module>
    sys.exit(load_entry_point('oci-env', 'console_scripts', 'oci-env')())
  File "/home/dkliban/devel/oci_env/client/oci_env/main.py", line 95, in main
    args.func(args, client)
  File "/home/dkliban/devel/oci_env/client/oci_env/commands.py", line 22, in db
    raise Exception(f'db {args.arction} not implemented')
AttributeError: 'Namespace' object has no attribute 'arction'. Did you mean: 'action'?
```

File: client/oci_env/commands.py [hunk 1]
```diff
@@ -11,7 +11,15 @@ def exec(args, client):
...
 
 def db(args, client):
-    pass
+    if args.action == 'reset':
+        exit_if_failed(
+            client.exec_container_script(
+                f"database_reset.sh",
+                args=None,
+                interactive=True)
+        )
```
Comment (newswangerd) [inline client/oci_env/commands.py:20]: adding `client.compose_command("restart")` will restart the services and you can get rid of the warning in the shell script

Comment (jctanner) [reply client/oci_env/commands.py:20]: done

**Review by dkliban — APPROVED** [review]

**Review by newswangerd — APPROVED** [review]

File: base/container_scripts/database_reset.sh [hunk 2]
```diff
@@ -1,14 +1,7 @@
 #!/bin/bash
 
-# this gives us the reset_db django command
-pip show django-extensions || pip install django-extensions
-
 # the operation will error out unless all services are stopped
 ls /var/run/s6/services | egrep ^pulp | xargs -I {} s6-svc -d /var/run/s6/services/{}
 
 # wipe the database non-interactive
 yes yes | pulpcore-manager reset_db --user postgres
```
Comment (mdellweg) [inline base/container_scripts/database_reset.sh:7]: I think django-admin (and as result pulpcore-manager) supports `--no-input` which could have been named `--force`.

**Pull request merged**